README.md

```diff
@@ -59,22 +122,31 @@ The `adminPasswordOrKey` parameter in this example command uses an existing SSH
 When the ARM template completes successfully you will have an AKS cluster in the provided resource group.
 
 ```sh
-az aks list --resource-group <resource-group>
+az aks list --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME
 ```
 
-Find the cluster name in the command output
+Export the AKS cluster name to a environment variable:
+
+```sh
+export OAM_TUTORIAL_AKS_CLUSTER_NAME=$(az aks list --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME | jq -r '.[0].name')
+```
 
 ```sh
-az aks get-credentials --name <cluster-name> --resource-group <resource-group>
+az aks get-credentials --name $OAM_TUTORIAL_AKS_CLUSTER_NAME --resource-group $OAM_TUTORIAL_RESOURCE_GROUP_NAME
 ```
 Now you should have credentials in your Kubernetes configuration for the AKS cluster. Your workloads will be deployed in this Kubernetes cluster.
 
 ### Connect to the control plane VM
 
+Extract the VM's *public* IP address:
+```
+export OAM_TUTORIAL_VM_IP=$(az vm list-ip-addresses -g $OAM_TUTORIAL_RESOURCE_GROUP_NAME | jq -r '.[0].virtualMachine.network.publicIpAddresses[0].ipAddress')
```

Comment:
```
--query "[0].virtualMachine.network.publicIpAddresses[0].ipAddress" --output tsv
```

Reply:
OH I just noticed that the credential management script uses JQ as well. I was trying to remove it as a dependency but I think that will have to wait until we get rid of remote mode.